meadServer/client/src/App.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Instead of 
```
     var batchNamesJSON = [];
     var i = 0;
     for (let name of this.state.batchNames)
     {
       batchNamesJSON.push({value: i, label: name});
       i++;
     }
```

`batchNamesJSON = this.state.batchNames.map((name, index) = >{ label: name, value: index })`